src/components/messenger/chat/index.tsx

```diff
@@ -37,16 +38,19 @@ export class Container extends React.Component<Properties, State> {
 
   static mapState(state: RootState): Partial<Properties> {
     const {
+      authentication,
```

Comment:
I think we should start using the existing selectors for this:
`const currentUser = currentUserSelector(state);`

I know most of the other places reach all the way into the authentication object but that makes things very inflexible.